platform/security/sts/security-sts-propertyclaimshandler/src/main/java/org/codice/ddf/security/sts/claims/property/PropertyFileClaimsHandler.java

```diff
@@ -160,7 +169,30 @@ public void setPropertyFileLocation(String propertyFileLocation) {
     if (propertyFileLocation != null
         && !propertyFileLocation.isEmpty()
         && !propertyFileLocation.equals(this.propertyFileLocation)) {
-      userMapping = PropertiesLoader.toMap(PropertiesLoader.loadProperties(propertyFileLocation));
+
+      Properties p = new Properties();
```

Comment:
✏️ Consider using a better variable name like `props` or `properties`

Reply:
Thanks, changed to props